### PR TITLE
Heyeman.tests

### PR DESCRIPTION
**Pull request by Heyeman**

### CODE REVIEW

**Comment by GedionT** [comment]
Can we follow this pattern for creating custom error with status codes


## first import the custom error model
 import { CustomError } from '../common/errorModel'

## then throw the error in (errMsg, statusCode, extraInfo)
 throw new CustomError('Oh no, this is embarrassing. We are having internal troubles', 404, 'some additional info')

File: starter-project-backend-g33/src/resources/articles/controller.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ const getArticle = (req: Request, res: Response, next: NextFunction) => {
     .getOne(req.params['id'])
     .then((data: any) => {
       if (!data) {
+        res.status(404)
```
Comment (GedionT) [inline starter-project-backend-g33/src/resources/articles/controller.ts:24]: I've dropped an update for all the custom errors we throw.

**Review by GedionT — CHANGES_REQUESTED** [review]

File: starter-project-backend-g33/src/resources/articles/controller.ts (hunk 1 above)
Comment (Heyeman) [reply starter-project-backend-g33/src/resources/articles/controller.ts]: I have added the custom errors as per the examples you dropped above.

Comment (GedionT) [reply starter-project-backend-g33/src/resources/articles/controller.ts]: Great, will review soon and merge

**Review by GedionT — APPROVED** [review]
everything looks okay, I have fixed the conflicts

**Pull request merged**